src/main/java/org/jenkinsci/plugins/parallel_test_executor/InclusionExclusionPattern.java

```diff
@@ -11,7 +11,7 @@
  */
 public class InclusionExclusionPattern implements Serializable {
     @Whitelisted
-    public boolean isIncludes() {
+    public boolean getIncludes() {
```

Comment:
As a name, `getIncludes` smells wrong (too - assuming it was changed because you felt `isIncludes` is wrong). Maybe a javadoc would clarify what it's supposed to be.